notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>Does stopping at the optimal point save resources used in running the tests? If yes, we should emphasize that here. Also, let's say we find the the optimal point, do we have a method to measure the impact of stopping at that point? It seems like it can save a ton of resources but we should think about how we can show that through an example. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Yes, the aim of finding the optimal stopping point is to save resources. We will add it to this cell :) </p><blockquote>do we have a method to measure the impact of stopping at that point?</blockquote><p>We have not really thought about that but definitely will look into this as a future task for our model.</p>